### ISSUE TITLE
DAC 6 - High Priority WCAG Level A - Sensory Characteristics 01

### ISSUE THREAD

**Issue by JadeneAd** [ONSvisual/income-scrolly#6]
<img width="286" alt="DAC_Sensory_Characteristics_01" src="https://user-images.githubusercontent.com/75850106/155550124-b77114d8-55db-4640-b2d7-c2d38b81b6ab.png">

### Issue
Users that navigate to the first piece of given information for a map will find that they are instructed to view content that is visually ‘In red’. Blind screen reader users may be unable to find the content that is distinguished by a red colour, and a similar experience applies for those that have difficulty perceiving colour. There are similar instances of this issue throughout the page.

<img width="601" alt="DAC 6 Current code ref" src="https://user-images.githubusercontent.com/75850106/155550344-6cb3fc7d-a770-44ef-981f-34b1a52562e4.png">

### Suggested Solution:
Instructing users to view content with the use of colour is not advised. In this instance, we recommend listing the 4 neighbourhoods that were among the 20 percent most income- deprived.
